Controller/Adminhtml/Hub/Index.php

```diff
@@ -101,6 +101,61 @@ private function getWebHookkUrl()
         return $baseUrl . "rest/V1/pagarme/webhook";
     }
 
+    private function allWebsitesIntegrated($currentWebsiteId)
+    {
+        $websites = $this->storeManager->getWebsites();
+        $magento2CoreSetup = new Magento2CoreSetup();
+
+        foreach ($websites as $websiteId => $website) {
+            $storeId = $this->storeManager->getWebsite($websiteId)
+                ->getDefaultStore()->getId();
+
+            $this->storeManager->setCurrentStore($storeId);
+
+            $magento2CoreSetup->loadModuleConfigurationFromPlatform();
+            $actualConfigurations = Magento2CoreSetup::getModuleConfiguration();
```

Comment:
~~actualConfigurations~~ -> currentConfigurations

actual = real
current = atual

Outra dúvida que fiquei, se não me engano temos utilizado esse método estático `Magento2CoreSetup::getModuleConfiguration` recentemente em alguns pontos da integração com o Hub. Porém por ser estático, ele não traz dados por instância, então se houverem várias compras sendo processadas, ou outros processos, de outras lojas, não necessariamente serão retornadas as configurações corretas por loja, por exemplo, mesmo que tenhamos acabado de executar o método o `loadModuleConfigurationFromPlatform`. 

Não pensei esse ponto antes pra comentar, mas acho que pode ser um ponto pra pensarmos se faz sentido e se pode gerar algum problema (ou se estou viajando hehe). Se for isso mesmo talvez teremos que pensar em outras abordagens, ver como foi feito no Magento 1, etc.
@vberardi vê também o que você acha.